janusgraph-all/pom.xml

```diff
@@ -62,6 +67,16 @@
             <groupId>org.janusgraph</groupId>
             <artifactId>janusgraph-es</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.ow2.asm</groupId>
+                    <artifactId>asm</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.ow2.asm</groupId>
+                    <artifactId>asm-commons</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
This is to resolve a dependency conflict between `org.elasticsearch:elasticsearch` and ``com.datastax.cassandra:cassandra-driver`. Both of these depend on `org.ow2.asm:asm`.
Elasticsearch depends on version 4.1 whereas the Cassandra driver depends on 5.0.3.
I didn't feel that dependency managing it in the parent pom was the right solution as this issue only comes up when both janusgraph-es and janusgraph-cql are included as dependencies.
The cleanest resolution in my opinion is to select the more recent version by excluding the version from elasticsearch.
Another alternative would be to explicitly depend in `asm` at the correct version but we'd have to manage that version manually.